### PR TITLE
World Object Fix

### PR DESCRIPTION
**Pull request by Forbiddenz**

### CODE REVIEW

File: Source/ACE/Network/Enum/Useability.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿namespace ACE.Network.Enum 
+{
+    public enum Useability : uint
+    {
+        USEABLE_UNDEF                                           = 0,
```
Comment (Mogwai-AC) [inline Source/ACE/Network/Enum/Useability.cs]: Convert to hex and sort by number please.  Also switch to ProperCase for element names.

File: Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs [hunk 2]
```diff
@@ -44,7 +44,7 @@ public GameMessageCreateLifestone(Player player) : base(GameMessageOpcode.Object
             wo.GameData.Type = (ushort)0x1355;
             wo.GameData.Icon = (ushort)0x1036;
 
-            wo.GameData.Useability = (uint)32;
+            wo.GameData.Useability = Useability.USEABLE_REMOTE;
```
Comment (Mogwai-AC) [inline Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs]: Usability*

File: Source/ACE/Entity/GameData.cs [hunk 3]
```diff
@@ -22,7 +22,7 @@ public class GameData
         public byte ContainerCapacity;
         public AmmoType AmmoType;
         public uint Value;
-        public uint Useability = 1u;
+        public Useability Useability = Useability.USEABLE_NO;
```
Comment (Mogwai-AC) [inline Source/ACE/Entity/GameData.cs]: Usability*

File: Source/ACE/Network/Enum/Useability.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
+﻿namespace ACE.Network.Enum 
+{
+    public enum Useability : uint
```
Comment (Mogwai-AC) [inline Source/ACE/Network/Enum/Useability.cs]: Please correct spelling of enum and file name.

**Comment by Forbiddenz** [comment]
Fixed,

**Pull request closed without merging**